packages/strapi/bin/strapi-install.js

```diff
@@ -77,7 +77,7 @@ module.exports = function (plugin, cliArguments) {
           fs.accessSync(path.join(pluginPath, '.gitignore'))
         } catch (err) {
           if (err.code === 'ENOENT') {
-            fs.copySync(path.resolve(__dirname, '..', '..', 'strapi-generate-plugin', 'files', '.gitignore'), path.join(pluginPath, '.gitignore'));
+            fs.copySync(path.resolve(__dirname, '..', 'node_modules', 'strapi-generate-plugin', 'templates', 'gitignore'), path.join(pluginPath, '.gitignore'));
```

Comment:
Do we still need this line? Because it has been replaced by your changes https://github.com/strapi/strapi/pull/405/files#diff-ac753172f10845a4618a791552b941c9

Reply:
Yes, strapi install use this line and strapi generate plugin use the copy function